js/build.ts

```diff
@@ -4,30 +4,26 @@
 /** Build related information */
 export interface BuildInfo {
   /** The operating system CPU architecture. */
-  arch: "x64";
+  arch: "x64" | "arm64";
```

Comment:
We should probably create a type for arch as well?

```ts
export type Architecture = "x64" | "arm64";
```

Reply:
done